Panic while processing helmfile hook

```
❯ helmfile version
helmfile version v0.126.0
```

Running into a panic when helmfile is trying to process hooks in the helmfile.yaml

```
helmfile.yaml: basePath=.
hook[./scripts/download-crds.sh]: triggered by event "prepare"

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x137d6de]

goroutine 104 [running]:
go.uber.org/zap.(*SugaredLogger).log(0x0, 0xff, 0x2196689, 0x4, 0xc00021fe20, 0x2, 0x2, 0x0, 0x0, 0x0)
        /Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/go.uber.org/zap@v1.9.1/sugar.go:221 +0x17e
go.uber.org/zap.(*SugaredLogger).Debugf(...)
        /Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/go.uber.org/zap@v1.9.1/sugar.go:133
github.com/roboll/helmfile/pkg/helmexec.(*logWriter).Write(0xc000521080, 0xc000690000, 0x30, 0x8000, 0x30, 0x0, 0x0)
        /private/tmp/helmfile-20200829-71424-cti1h7/helmfile-0.126.0/pkg/helmexec/log.go:25 +0x15f
io.(*multiWriter).Write(0xc0005210c0, 0xc000690000, 0x30, 0x8000, 0x30, 0x0, 0x0)
        /usr/local/Cellar/go/1.15/libexec/src/io/multi.go:60 +0x87
io.copyBuffer(0x25d6080, 0xc0005210c0, 0x25d6520, 0xc0002cd380, 0xc000690000, 0x8000, 0x8000, 0x0, 0xc00021ffa8, 0x1008345)
        /usr/local/Cellar/go/1.15/libexec/src/io/io.go:411 +0x206
io.Copy(...)
        /usr/local/Cellar/go/1.15/libexec/src/io/io.go:368
os/exec.(*Cmd).writerDescriptor.func1(0x21f3730, 0xc00021ffc8)
        /usr/local/Cellar/go/1.15/libexec/src/os/exec/exec.go:311 +0x65
os/exec.(*Cmd).Start.func1(0xc000160160, 0xc000521200)
        /usr/local/Cellar/go/1.15/libexec/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
        /usr/local/Cellar/go/1.15/libexec/src/os/exec/exec.go:440 +0x629
```

--- a/pkg/event/bus.go
+++ b/pkg/event/bus.go
@@ -41,8 +41,8 @@ type Bus struct {
 func (bus *Bus) Trigger(evt string, evtErr error, context map[string]interface{}) (bool, error) {
 	if bus.Runner == nil {
 		bus.Runner = helmexec.ShellRunner{
-			Dir: bus.BasePath,
-			//Logger: bus.Logger,
+			Dir:    bus.BasePath,
+			Logger: bus.Logger,
 		}
 	}
 

--- a/pkg/event/bus.go
+++ b/pkg/event/bus.go
@@ -42,6 +42,7 @@ func (bus *Bus) Trigger(evt string, evtErr error, context map[string]interface{}
 	if bus.Runner == nil {
 		bus.Runner = helmexec.ShellRunner{
 			Dir: bus.BasePath,
+			//Logger: bus.Logger,
 		}
 	}
 

--- a/test/integration/happypath.yaml
+++ b/test/integration/happypath.yaml
@@ -19,6 +19,14 @@ releases:
     set:
       - name: ingress.enabled
         value: false
+    hooks:
+    - events: ["prepare", "cleanup"]
+      showlogs: true
+      command: "echo"
+      args:
+      - "{{`{{.Environment.Name}}`}}"
+      - "{{`{{.Release.Name}}`}}"
+      - "{{`{{.HelmfileCommand}}`}}"
 
   - name: raw
     chart: incubator/raw

